Source/Core/UICommon/Disassembler.cpp

```diff
@@ -172,7 +172,7 @@ std::string DisassembleBlock(HostDisassembler* disasm, u32* address, u32* host_i
 	}
 	else if (res == 2)
 	{
-		host_instructions_count = 0;
+		*host_instructions_count = 0;
```

Comment:
I gave this its own special commit, because this is a change that does something completely different from the original code, but I think this was the intent in the first place, rather than setting it to nullptr